mono/mini/objects.cs

```diff
@@ -1389,6 +1389,7 @@ sealed class Bar2 : Foo2 {
 		return 1.4e-45f;
 	}
 
+#if !NO_BITCODE
 	[Category ("!BITCODE")] // bug #59953
```

Comment:
Why is this disabled if it already has a !BITCODE category ?

Reply:
It was crashing the AOT compiler, I'll remove it once its fixed.